docs/config/node.rst

```diff
@@ -746,6 +746,26 @@ Queries
     ``ANY`` operators on arrays of that length can lead to heavy memory,
     consumption which could cause nodes to crash with OutOfMemory exceptions.
 
+Legacy
+=======
+
+.. _legacy.table_function_column_naming:
+
+**legacy.table_function_column_naming**
+  | *Default:* ``false``
+  | *Runtime:* ``no``
+
+  Since CrateDB 5.0.0, if the table function is not aliased and is returning a
+  base data type, the table function name is used as the column name. This
+  setting can be set in order to use the naming convention prior to 5.0.0.
+
+  .. NOTE::
+
+    To be precise, the table functions affected are ``unnest``,
+    ``regexp_matches``, and ``generate_series`` which will return the output
+    column names as ``col1`` when set. Any other table functions defined after
```

Comment:
`regexp_matches` returns `groups` instead of `col1` in legacy mode, no?

Reply:
Yes, will fix this. Thanks :+1: